pkg/common/utils/resource/pod.go

```diff
@@ -500,8 +500,31 @@ func buildEnvFromPod() []corev1.EnvVar {
 	}
 }
 
-func GetPodDefaultEnv() []corev1.EnvVar {
-	return buildEnvFromPod()
+func GetPodDefaultEnv(ddc *dv1.DorisDisaggregatedCluster) []corev1.EnvVar {
+	defaultEnvs := buildEnvFromPod()
+
+	if ddc.Spec.AdminUser != nil {
+		defaultEnvs = append(defaultEnvs, corev1.EnvVar{
+			Name:  ADMIN_USER,
+			Value: ddc.Spec.AdminUser.Name,
+		})
+		if ddc.Spec.AdminUser.Password != "" {
+			defaultEnvs = append(defaultEnvs, corev1.EnvVar{
+				Name:  ADMIN_PASSWD,
+				Value: ddc.Spec.AdminUser.Password,
+			})
+		}
+	} else {
+		defaultEnvs = append(defaultEnvs, []corev1.EnvVar{{
```

Comment:
This behavior will trigger the restart of the DORIS cluster after upgrading the operator. It is dangerous for the online environment.

Reply:
Done. Currently, the doris cluster will only be restarted when `ddc.Spec.AdminUser` is configured.